package/network/utils/iwinfo/patches/0015-nl80211-fix-frequency-channel-conversion-for-the-6G-.patch

```diff
@@ -0,0 +1,101 @@
+From fad47e19511aee0d5ada55fdce85075236f2a9ee Mon Sep 17 00:00:00 2001
+From: Andre Heider <a.heider@gmail.com>
+Date: Thu, 17 Nov 2022 21:02:16 +0100
+Subject: [PATCH 15/39] nl80211: fix frequency/channel conversion for the 6G
+ band
+
+Update using current linux sources, which this was apparently
+based on.
```

Comment:
If this is copied from some other implementation please mention this in the file.
I think we should also preserve the copyright statements.

Reply:
I did mention it in as a comment, but right you are, added the license and copyright statements too